generator/model.go

```diff
@@ -543,8 +543,20 @@ func hasValidations(model *spec.Schema, isRequired bool) (needsValidation bool,
 	return
 }
 
+// handleFormatConflicts handles all conflicting model properties when a format is set
+func handleFormatConflicts(model *spec.Schema) {
+	switch model.Format {
+	case "date", "datetime", "uuid":
```

Comment:
also: bsonobjectid, base64 and duration

none of those are actually strings so the validations don't apply to them